tests/integration-tests/tests/performance_tests/test_startup_time.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import statistics
+from datetime import datetime
+
+import boto3
+import pytest
+from dateutil.relativedelta import relativedelta
+from utils import describe_cluster_instances
+
+MINIMUM_DATASET_SIZE = 2
```

Comment:
Why a sample size of 2. 
Is it possible to increase the dataset size to have more statistically powerful insights?

Reply:
I will increase it to 5